tests/test_boundaries.py

```diff
@@ -68,25 +67,48 @@ def test_anti_bounce_back_outlet(f_lattice):
         if Q == 27:
             u_w = u[:, -1, :, :] + 0.5 * (u[:, -1, :, :] - u[:, -2, :, :])
             for i in [1, 11, 13, 15, 17, 19, 21, 23, 25]:
-                f_ref[lattice.stencil.opposite[i], -1, :, :] = - f_ref[i, -1, :, :] + lattice.stencil.w[i] * lattice.rho(f)[0, -1, :, :] * \
-                    (2 + torch.einsum('c, cyz -> yz', torch.tensor(lattice.stencil.e[i], device=f.device, dtype=f.dtype), u_w) ** 2 / lattice.stencil.cs ** 4 - (torch.norm(u_w, dim=0) / lattice.stencil.cs) ** 2)
+                f_ref[lattice.stencil.opposite[i], -1, :, :] = - f_ref[i, -1, :, :] + lattice.stencil.w[
```

Comment:
Maybe there is a way that is not that ugly but also satisfies pep...

Reply:
Yes, please ;-)
Maybe just open parentheses behind the `=` and break lines for each summand.